lib/rollbax/item.ex

```diff
@@ -27,7 +27,7 @@ defmodule Rollbax.Item do
     if map_size(meta) == 0 do
       data
     else
-      Map.put(data, "custom", meta)
+      Map.merge(data, meta)
```

Comment:
> You're talking about something like `Rollbax.report(..., %{my_thing: 123, root_data: %{person: %{id: 123}}})` which would result in `%{env: "as_usual", person: %{id: 123}, custom: %{my_thing: 123}}` to be reported to rollbar?

Exactly.

> also, i don't see the problem with `uuid`.

The problem with `uuid` is if I'm, as a user, calling `Rollbax.report(.., .., %{uuid: "xyz"})` I'm expecting this `uuid` will be treated as metadata (it just could be UUID of some record in the DB) but in case of merging `meta` with `data` it'll be taken as `uuid` of an error occurrence, and might be rewritten on next call with same `uuid`.

Let's go this way:
```elixir
def report(exception, stacktrace, meta \\ %{} , occurr_data \\ %{})
when is_list(stacktrace) and is_map(meta) and is_map(occurr_data) do
  message = Exception.format(:error, exception, stacktrace)
  meta = Map.put(meta, :rollbax_occurr_data, occurr_data)
  Rollbax.Client.emit(:error, message, meta)
end
```
* make `Rollbax.report/3` –> `Rollbax.report/4` which treats last argument as extra occurrence data.